package.json

```diff
@@ -20,14 +20,15 @@
         "laravel-mix": "^6.0.12",
         "lodash": "^4.17.20",
         "popper.js": "^1.16.1",
+        "postcss": "^8.2.8",
         "resolve-url-loader": "^2.3.1",
         "sass": "^1.26.10",
         "sass-loader": "^7.3.1",
```

Comment:
When testing I had to bump these up to:
```
        "resolve-url-loader": "^3.1.2",
        "sass": "^1.32.12",
        "sass-loader": "^11.0.1",
```

Reply:
I've run yarn one time and all passed. 🤔
Possibly an optional bump!?

Reply:
I've cleaned the npm dependencies up a bit. But I'm unable to build the assets with `npm run dev` - `yarn dev` works just fine. 🤔🤯
It's not related to my dependency changes - was the same before.
https://github.com/tighten/onramp/pull/301/commits/59e463fcaa575be6e639124d9b2b36563afc31a2

The error I get is:

```
ERROR in ./resources/js/app.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: validateOptions is not a function
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:71:5)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (node_modules/babel-loader/lib/index.js:3:103)
    at _next (node_modules/babel-loader/lib/index.js:5:194)
    at node_modules/babel-loader/lib/index.js:5:364
    at new Promise (<anonymous>)
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:5:97)
    at Object._loader (node_modules/babel-loader/lib/index.js:235:18)
    at Object.loader (node_modules/babel-loader/lib/index.js:64:18)
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:59:12)

ERROR in ./resources/js/scripts.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: validateOptions is not a function
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:71:5)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (node_modules/babel-loader/lib/index.js:3:103)
    at _next (node_modules/babel-loader/lib/index.js:5:194)
    at node_modules/babel-loader/lib/index.js:5:364
    at new Promise (<anonymous>)
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:5:97)
    at Object.loader (node_modules/babel-loader/lib/index.js:64:18)
    at Object.<anonymous> (node_modules/babel-loader/lib/index.js:59:12)

webpack compiled with 2 errors
```

I have absolutely no idea why/what and as yarn works just fine it could also be my local `node/npm` environment.

Reply:
I pulled down your latest changes and was able to `npm install && npm run dev` and `yarn install && yarn run dev` removing `node_modules` before each run.

Here's what I'm running if it helps: 
- Node: 16.0.0
- NPM: 7.10.0
- Yarn: 1.22.10

Reply:
If it works for you I'm fine. 🙂
I was able to run it with `yarn` - so I think that everything is okay!?